### ISSUE TITLE
ImportOrder reports incorrect errors for redundant imports

### ISSUE THREAD

**Issue by cushon** [checkstyle/checkstyle#6303]
```java
import java.util.List;
import java.util.List;

public class T {
  private List<?> l;
}
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC
    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="RedundantImport" />
    <module name="ImportOrder" />
  </module>
</module>
```
## Actual

```shell
java -jar checkstyle-8.15-all.jar -c config.xml T.java
Starting audit...
[ERROR] T.java:2: Wrong order for 'java.util.List' import. [ImportOrder]
[ERROR] T.java:2:1: Duplicate import to line 1 - java.util.List. [RedundantImport]
Audit done.
Checkstyle ends with 2 errors.
```

## Expected

The `ImportOrder` finding is incorrect: the two imports of `java.util.List` are identical, and are not out of order. The real issue is the one reported by `RedundantImport`.

**Comment by rnveach** [comment]
@cushon Why do you say the redundant import violation is the issue? Violation looks correct to me. Both imports are exactly the same and only 1 is needed.

**Comment by cushon** [comment]
There was a typo. I was trying to say that the `ImportOrder` finding is unnecessary, since the list of imports is sorted.

**Comment by rnveach** [comment]
I agree that import order shouldn't complain about the same import twice. That is redundant's job.

**Comment by romani** [comment]
Fix is merged
